packages/babel-preset-expo/src/__tests__/__snapshots__/compiler.test.ts.snap

```diff
@@ -3,7 +3,7 @@
 exports[`allows destructuring in the catch block 1`] = `
 "Object.defineProperty(exports, "__esModule", { value: true });exports.App = App;var _jsxRuntime = require("react/jsx-runtime");function App() {
   // eslint-disable-next-line @typescript-eslint/no-unused-vars
-  var foo = function () {
+  var foo = () => {
```

Comment:
This is a pretty big change, did Hermes update and add support for this functionality?

Reply:
Not a ton of info here https://github.com/facebook/react-native/pull/44534

Reply:
Thanks for digging into root of this! 🙏